script.js

```diff
@@ -11,3 +11,108 @@ hamburger.addEventListener('click', () => {
+  projectLink: 'https://github.com/Mikey1-nuel/Setup-and-Mobile-First',
+};
+
+const headerTitle = document.querySelector('.header1');
+const headerImg = document.querySelector('.image1');
+const headerDescription = document.querySelector('paragraph1');
+const headerLanguages = document.querySelector('.lang-list1');
+let language = '';
+mainProject.headerTitle.forEach((langcontainer) => {
+  language = `${language}<li>${langcontainer}<li>`;
+});
+headerTitle.innerHTML = mainProject.headerTitle;
+headerImg.src = `images/${mainProject.headerImg}`;
+headerDescription.innerHTML = mainProject.headerDescription;
+headerLanguages.innerHTML = language;
+
+const workCards = [{
+  cardId: 1,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 2,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 3,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 4,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 5,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 6,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+];
+
+const projectCards = document.querySelector('.main-container');
+projectCards.innerHTML = '';
+for (let i = 0; i < workCards.length; i += 1) {
+  let language = '';
+  workCards[i].cardLanguages.forEach((cardLanguages) => {
+    language = `${language}<li>${cardLanguages}<li>`;
+  });
+  projectCards.innerHTML += `<div class="container3">
+  <h1 class="show-text1">${workCards[i].cardTitle}</h1>
+  <p class="paragraph4">${workCards[i].cardDescription}</p>
+  <ul class="lang-list2">${language}</ul>
+  <button class="btn1 display" type="button">See Project</button>
+  </div>
+  `;
```

Comment:
- [ ] I will implore you to make the header and close button show on the popup. At the moment, I can not even close the popup because no close sign is showing.
- [ ] Also, you can make it scrollable so that the buttons under too will show.
### Your Design
![image](https://user-images.githubusercontent.com/86200356/225268928-a4454d3d-9000-4bce-a1cc-7f4ca1960161.png)
### Figma Design
![image](https://user-images.githubusercontent.com/86200356/225269617-fa2aa1f1-549a-440d-9080-4982b351dc35.png)